service/authentication_validation.go

```diff
@@ -32,6 +33,10 @@ func ValidateAuthenticationCreationRequest(auth *model.AuthenticationCreateReque
 	// capitalize it so it's always the same format.
 	auth.ResourceType = util.Capitalize(auth.ResourceType)
 
+	if err := ValidateAzureSubcriptionId(auth); err != nil{
```

Comment:
Since the `err` variable is already defined we can simply reassign it with `=` instead of redeclaring it with `:=`.

```suggestion
	if err = ValidateAzureSubscriptionId(auth); err != nil{
```